website/server/libs/inbox/index.js

```diff
@@ -1,12 +1,25 @@
 import { inboxModel as Inbox } from '../../models/message';
 
-export async function getUserInbox (user, asArray = true) {
-  const messages = (await Inbox
+const PM_PER_PAGE = 10;
+
+export async function getUserInbox (user, options = {asArray: true, page: 0}) {
+  if (typeof options.asArray === 'undefined') {
```

Comment:
why check if the option is undefined if we already provide a default in the parameter definition?

Reply:
if you call it like ```
const userInbox = await inboxLib.getUserInbox(user, {
      page,
    });
```

then `asArray` won't be set even it is available as the default parameter thats why the second  check is needed